src/main/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/ERICHeaderParser.java

```diff
@@ -27,4 +31,33 @@ public String getEmailAddress(String ericAuthorisedUser) {
 
         return email;
     }
+
+    public String getFullName(String ericAuthorisedUser) {
+        AtomicReference<String> foreName = new AtomicReference<>();
```

Comment:
Either 'forename' or change to 'surName' for consistency. Would favour the former.